integration-tests/config/testplan.json

```diff
@@ -0,0 +1,25 @@
+{
+    "templates": [
+        "go"
+    ],
+    "tssc": [
+        {
+            "git": "github",
+            "ci": "tekton",
+            "registry": "quay.io",
+            "tpa": "local",
+            "acs": "remote"
```

Comment:
In `rhads-config` file is `ACS="local"` and here is `"acs": "remote"`. What does that mean?

Reply:
`local` means the component is installed by rhtap-cli on the same cluster. This is to test rhtap-cli is able to install the component properly.
`remote` means the component is installed on different cluster from RHADS.